### PR TITLE
Complete test cases for the scenario in which there is only 1 and 2 page of result, validate the result and pagination

### PR DESCRIPTION
**Pull request by nthikimngoc**

### CODE REVIEW

File: src/homework-NN.js
Comment (liamnguyeniaa) [inline src/homework-NN.js:1]: review the comments left on Nhi's PR and add something different than Nhi's approach.

**Review by liamnguyeniaa — CHANGES_REQUESTED** [review]
comments

File: src/02-so2-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+test("search for 'giới thiệu' and only 1 page of result, validate the result and pagination", async (t) => {
+    const searchInputSelector = Selector(".search .input-group .search");
+       
+    await t.expect(searchInputSelector.exists).ok("Search box should exist");
+  
+    await t.typeText(searchInputSelector, "giới thiệu").pressKey("enter");
+  
+    await t
+      .expect(Selector(".main .thong-tin .blog-details .btn").exists)
+      .ok("At more six search result should exist");
```
Comment (liamnguyeniaa) [inline src/02-so2-ttts2023-agu-pagination-ngocnguyen.test.js]: incorrect message. 

need to validate that the pagination doesn't appear.

File: src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+//Write test cases for the scenario in which there are 2 pages of result, validate the result and pagination
+
+import { Selector } from "testcafe";
+
+fixture("ttts2023-agu").page("https://www.agu.edu.vn/vi/tim-kiem");
+
+test("search for 'Công khai' and there are 2 pages of result, validate the result and pagination", async (t) => {
+  const searchInputSelector = Selector(".search .input-group .search");
+  const ClickNextPage = Selector(".main .page-item:last-child .page-link");
```
Comment (liamnguyeniaa) [inline src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js]: incorrect variable name. use camelCase.

File: src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+  const ClickPreviousPage = Selector(".main .page-item:first-child .page-link");
+ 
+  await t.expect(searchInputSelector.exists).ok("Search box should exist");
+
+  await t.typeText(searchInputSelector, "Công khai").pressKey("enter");
+
+  await t
+    .expect(Selector(".main .thong-tin .blog-details .btn").exists)
+    .ok("At more six search result should exist");
```
Comment (liamnguyeniaa) [inline src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js]: incorrect message

File: src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+
+  for (let i = 0; i < 5; i++) {
```
Comment (liamnguyeniaa) [inline src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js]: there are only 2 pages. Why do we need to click 5 times?

File: src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      await t.click(ClickNextPage);
+    }
+    
+  await t.expect(ClickNextPage.exists).ok("Next page search should exist");
+
+  await t.expect(ClickPreviousPage.exists).ok("Previous page search should exist");
+
+  await t.click(ClickNextPage);
+
+  await t.click(ClickPreviousPage);
```
Comment (liamnguyeniaa) [inline src/02-so3-ttts2023-agu-pagination-ngocnguyen.test.js]: the test case should never end with action.

File: src/02-ttts2023-agu-pagination-ngocnguyen.test.js [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+  await t.typeText(searchInputSelector, "tuyen sinh").pressKey("enter");
+
+  await t
+    .expect(Selector(".main .thong-tin .blog-details .btn").exists)
+    .notOk("At least one search result should exist");
+});
+
+test("search for 'tuyển sinh' and validate some results appear", async (t) => {
+  const searchInputSelector = Selector(".search .input-group .search");
+  const ClickNextPage = Selector(".main .page-item:last-child .page-link");
```
Comment (liamnguyeniaa) [inline src/02-ttts2023-agu-pagination-ngocnguyen.test.js]: incorrect variable name.

**Review by liamnguyeniaa — COMMENTED** [review]
1. Review variable naming
1. make sure the test case never end with action
2. fix all the comments.

File: src/02-ttts2023-agu-pagination-ngocnguyen.test.js (hunk 6 above)
Comment (nthikimngoc) [reply src/02-ttts2023-agu-pagination-ngocnguyen.test.js]: Done

Comment (nthikimngoc) [reply src/02-ttts2023-agu-pagination-ngocnguyen.test.js]: Done

**Pull request merged**